generic_config_updater/change_applier.py

```diff
@@ -150,6 +157,13 @@ def apply(self, change):
         return ret
 
 
+    def get_config_without_backend_tables(self, data):
+        cropped_data = {
+            k: v for k, v in data.items() if k not in self.backend_tables
+        }
+        return cropped_data
```

Comment:
I don't see a reason to iterate over the entire database to remove a specific list of keys, we can directly remove the tables that should not be tested.

Something like this:
```suggestion
    def get_config_without_backend_tables(self, data):
        for key in self.backend_tables:
            data.pop(key, None)
```

Reply:
Modified. Please review again. Thanks